cpp/src/arrow/util/compression_test.cc

```diff
@@ -62,20 +62,15 @@ void CheckCodecRoundtrip(std::unique_ptr<Codec>& c1, std::unique_ptr<Codec>& c2,
 
   // compress with c1
   int64_t actual_size;
-  ASSERT_OK(c1->Compress(data.size(), data.data(), max_compressed_len, compressed.data(),
-                         &actual_size));
+  ASSERT_OK_AND_ASSIGN(actual_size, c1->Compress(data.size(), data.data(),
```

Comment:
Ditto and the rest of the file.

Reply:
Not sure making the line longer will make the code more readable. Separate declarations can stick out.